### PR TITLE
410366 add scope param

### PR DESCRIPTION
**Pull request by matt-yin-lf**
- Added optional scope parameter
- Updated auto-generated token client
- Bumped the version to 1.3.1
- Updated the changelog for the optional scope parameter

### CODE REVIEW

File: src/OAuth/ITokenClient.cs [hunk 1]
```diff
@@ -24,9 +25,10 @@ public partial interface ITokenClient
         /// <param name="redirectUri">Authorization endpoint redirect URI.</param>
         /// <param name="clientId">OAuth application client ID.</param>
         /// <param name="clientSecret">OPTIONAL OAuth application client secret. Required for web apps.</param>
+        /// <param name="scope">(optional) The requested space-delimited scopes for the access token.</param>
         /// <param name="codeVerifier">OPTIONAL PKCE code verifier. Required for SPA apps.</param>
         /// <returns>A response that contains an access token.</returns>
-        Task<GetAccessTokenResponse> GetAccessTokenFromCode(string code, string redirectUri, string clientId, string clientSecret = null, string codeVerifier = null);
+        Task<GetAccessTokenResponse> GetAccessTokenFromCode(string code, string redirectUri, string clientId, string clientSecret = null, string scope = null, string codeVerifier = null);
```
Comment (jason-chan-lf) [inline src/OAuth/ITokenClient.cs]: should the scope be the last parameter? It is the last parameter in the implementation

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
 
 env:
-  VERSION_PREFIX: "1.3.0"
+  VERSION_PREFIX: "1.3.1"
```
Comment (jason-chan-lf) [inline .github/workflows/main.yml:12]: I think we might also have to bump the version in the `.csproj` file

File: src/OAuth/ITokenClient.cs (hunk 1 above)
Comment (matt-yin-lf) [reply src/OAuth/ITokenClient.cs]: Right, fixed.

File: .github/workflows/main.yml (hunk 2 above)
Comment (matt-yin-lf) [reply .github/workflows/main.yml:12]: Fixed.

**Review by jason-chan-lf — APPROVED** [review]

**Pull request merged**